hawc/apps/animal/models.py

```diff
@@ -476,37 +429,11 @@ class DosingRegime(models.Model):
 
     objects = managers.DosingRegimeManager()
 
-    ROUTE_EXPOSURE_CHOICES = (
-        ("OR", "Oral"),
-        ("OC", "Oral capsule"),
-        ("OD", "Oral diet"),
-        ("OG", "Oral gavage"),
-        ("OW", "Oral drinking water"),
-        ("I", "Inhalation"),
-        ("IG", "Inhalation - gas"),
-        ("IR", "Inhalation - particle"),
-        ("IA", "Inhalation - vapor"),
-        ("D", "Dermal"),
-        ("SI", "Subcutaneous injection"),
-        ("IP", "Intraperitoneal injection"),
-        ("IV", "Intravenous injection"),
-        ("IO", "in ovo"),
-        ("P", "Parental"),
-        ("W", "Whole body"),
-        ("M", "Multiple"),
-        ("U", "Unknown"),
-        ("O", "Other"),
-    )
-    ROUTE_EXPOSURE_CHOICES_DICT = {el[0]: el[1] for el in ROUTE_EXPOSURE_CHOICES}
-
-    POSITIVE_CONTROL_CHOICES = ((True, "Yes"), (False, "No"), (None, "Unknown"))
-
-    NEGATIVE_CONTROL_CHOICES = (
-        ("NR", "Not-reported"),
-        ("UN", "Untreated"),
-        ("VT", "Vehicle-treated"),
-        ("B", "Untreated + Vehicle-treated"),
-        ("N", "No"),
+    # TODO tried Class PositiveControl(int,models.Choices), but can't subclass bool?
+    POSITIVE_CONTROL_CHOICES = (
```

Comment:
bummer; let's move to constants anyways and keep this comment that we cant use the choices for booleans?